packages/extension/src/pages/sign/utils/handle-eth-sign.ts

```diff
@@ -56,13 +76,61 @@ export const handleEthereumPreSign = async (
       }
 
       return connectAndSignEthWithLedger(
-        useWebHID,
+        (options as LedgerOptions).useWebHID,
```

Comment:
@renfengshi I think it would be better to split the function. rather than `(options as LedgerOptions)`